### ISSUE TITLE
More space between nodes/edges

### ISSUE THREAD

**Issue by AKC0** [mingrammer/diagrams#615]
Hello, first of thanks for this Project! Unfortunatly I am only getting unclear outputs. In my case a lot of edges are going parallel which makes it hard to differ them. I didn't find a way to get more space between them. I did find Graphviz attributes like 'concentrate' or 'mindist' but they either just didn't do anything or even said:
"Warning: node 'xy', graph 'xy' size too small for label"
Thanks in advance and kind regards!

**Comment by AKC0** [comment]
![screenshot output](https://user-images.githubusercontent.com/93992266/151206681-8b36f51d-9c53-43ed-8307-3662aa5b0aac.PNG)
@clayms As you can see it is really hard to follow a line because they are too close together. I'd like to be able to set a minimum distance between parallel lines.

**Comment by clayms** [comment]
@AKC0  Please post your code.

Also have a look at the following:
https://github.com/mingrammer/diagrams/issues/579#issuecomment-899830574
https://github.com/mingrammer/diagrams/issues/579#issuecomment-898628367

and 
https://github.com/mingrammer/diagrams/issues/358#issuecomment-716565948
https://github.com/mingrammer/diagrams/issues/358#issuecomment-723581944
https://github.com/mingrammer/diagrams/issues/358#issuecomment-723674931

**Comment by clayms** [comment]
@AKC0 
You are not required to use "ortho",  change it to "spline" .

If you are using python to generate the code from the excel file, then you should be able to also modify the `Edge()`s dynamically.

**Comment by AKC0** [comment]
@clayms yeah but I want to use ortho 😄 As I said: I thought there would be the one attribute I could use but apparently there are only kludges so it's fine.
